pkg/subctl/cmd/cloud/prepare/aws.go

```diff
@@ -20,67 +20,49 @@ package prepare
 
 import (
 	"github.com/spf13/cobra"
-	"github.com/submariner-io/admiral/pkg/reporter"
-	"github.com/submariner-io/cloud-prepare/pkg/api"
 	"github.com/submariner-io/submariner-operator/internal/cli"
 	"github.com/submariner-io/submariner-operator/internal/exit"
+	"github.com/submariner-io/submariner-operator/internal/restconfig"
+	"github.com/submariner-io/submariner-operator/pkg/cloud"
+	cloudaws "github.com/submariner-io/submariner-operator/pkg/cloud/aws"
+	"github.com/submariner-io/submariner-operator/pkg/cloud/prepare"
 	"github.com/submariner-io/submariner-operator/pkg/subctl/cmd/cloud/aws"
 )
 
+var config *cloudaws.Config
```

Comment:
This needs to declare the instance itself, not just a pointer to it:

```suggestion
var config cloudaws.Config
```

Then whenever you need to pass `config` around, pass a pointer to it, `&config`.